src/content/docs/en/guides/integrations-guide/vercel.mdx

```diff
@@ -353,6 +353,26 @@ export default defineConfig({
 });
 ```
 
+### `skewProtection`
+
+**Type:** `boolean`<br/>
+**Available for:** Serverless
+
```

Comment:
```suggestion

<p><Since pkg="vercel" v="7.6.0" /></p>

```
I'm going to start remembering these the first edit round!  :sweat_smile:

Reply:
Ok. However, this **ins't** inline with the style of the other options. Do you still want to accept this change?